kapp_platform_common/src/events.rs

```diff
@@ -33,6 +33,16 @@ pub enum Event {
     CharacterReceived {
         character: char,
     },
+    /// The composition status of the IME.
```

Comment:
This is a nitpick, but generally (when I remember) I like to define acronyms the first time they appear in code or in a document.

Reply:
But considering these are doc comments, wouldn't it be a bit weird to document the acronym for `IMEComposition` but not for `IMEEndComposition`?